backend/game/cards/keep_cards/victory_point_manipulation_cards/urbavore.py

```diff
@@ -0,0 +1,20 @@
+from game.cards.card import Card
+from game.cards.keep_card import KeepCard
+from game.values import locations
+
+
+class Urbavore(KeepCard):
+    def __init__(self):
+        Card.__init__(self, "Urbavore", 4,
+                      "gain 1 extra [star] when beginning your turn in Tokyo. If your are in Tokyo and you roll at least one [attack] add [attack] to your roll",
```

Comment:
This card only is activated both on turn start and turn roll.  Both those conditions need to be met and a function needs to be created to not just add victory points but add attack if rolled.